crates/bevy_ecs/src/system/commands.rs

```diff
@@ -112,6 +112,24 @@ where
     }
 }
 
+pub(crate) struct Remove<T>
+where
+    T: Bundle + Send + Sync + 'static,
+{
+    entity: Entity,
+    phantom: PhantomData<T>,
+}
+
+impl<T> WorldWriter for Remove<T>
+where
+    T: Bundle + Send + Sync + 'static,
+{
+    fn write(self: Box<Self>, world: &mut World) {
+        // TODO: need equivalent of world.get::<T>(self.entity).is_ok() check from RemoveOne?
```

Comment:
short term lets just remove this comment and let it unwrap. medium term I'm considering making the remove_one/remove/despawn Commands swallow errors to allow for concurrent "removes".